decoder/vaapidecoder_base.cpp

```diff
@@ -346,6 +346,16 @@ Decode_Status
         INFO("setting LIBVA_DRIVER_NAME to wrapper for chromeos");
     }
 #endif
+
+    if (profile == VAProfileVP9Version0) {
```

Comment:
Playing VP9 and VP8 one after the other works fine. But playing VP8 and VP9 at the same time does not work. The reason is that for first stream, the vaDisplayContext stores the display context of i965 driver in vaDisplayContext->vaDriverContext. For the second stream, it will again use the context of i965 driver, even though the driver name is set as hybrid. Thus it will return 'profile not supported' error during vaCreateConfig(). I will file a JIRA on this issue, We need to figure out another way to solve this.